brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/accountverification/AccountVerificationViewModel.kt

```diff
@@ -43,15 +57,15 @@ class AccountVerificationViewModel(
 
             is AccountVerificationContract.Event.Level1Clicked ->
                 setEffect {
-                    when (currentState.profile.kycStatus) {
+                    when (currentState.profile?.kycStatus) {
                         KycStatus.DEFAULT,
                         KycStatus.EMAIL_VERIFIED,
                         KycStatus.EMAIL_VERIFICATION_PENDING,
                         KycStatus.KYC1,
                         KycStatus.KYC2_EXPIRED,
                         KycStatus.KYC2_DECLINED,
-                        KycStatus.KYC2_RESUBMISSION_REQUESTED ->
-                            AccountVerificationContract.Effect.GoToKycLevel1
+                        KycStatus.KYC2_RESUBMISSION_REQUESTED,
+                        null -> AccountVerificationContract.Effect.GoToKycLevel1
```

Comment:
Are we sure we want to navigate to KYC1 if profile is null? As far as I understand if profile is null it's still loading. We should probably wait until the profile is loaded before we can decide where to navigate?

Reply:
Hi, no we don't wont to. But user actually can't even click on KYC1 item, since the button is disabled and loading screen is shown to user. I was thinking to removing nullable from profile but then on opening screen when the profile would be null from arguments I should create a dummy profile which isn't any better. All considering just and to make sure in case the profile is null I've added a toast representing current condition. If I should handle this differently please let me know.